### ISSUE TITLE
Bug calling  a function by a package defined in a system loaded by quicklisp

### ISSUE THREAD

**Issue by ryukinix** [yitzchak/common-lisp-jupyter#10]
```lisp
(ql-dist:install-dist "http://dist.ultralisp.org" :prompt nil :replace t)
(ql:quickload :lisp-inference)
(inference:truth-infix (p ^ q => r))
```

# Expected

![image](https://user-images.githubusercontent.com/7642878/53463617-2a0a5800-3a26-11e9-9620-f398aac6d2e3.png)

# Actual behavior

![image](https://user-images.githubusercontent.com/7642878/53463644-3f7f8200-3a26-11e9-9d20-172496e44716.png)

fulltrace: http://ix.io/1C8u

**Comment by yitzchak** [comment]
Thanks for the feedback. I suspect this is related to Roswell's use of quicklisp. I'll look into it.

**Comment by yitzchak** [comment]
@ryukinix I think I have a fix for this in PR #11
